magicctapipe/scripts/lst1_magic/semi_automatic_scripts/DL2_to_DL3.py

```diff
@@ -0,0 +1,314 @@
+"""
+This script creates the bashscripts necessary to apply "lst1_magic_dl2_to_dl3.py"
+to the DL2. It also creates new subdirectories associated with
+the data level 3.
+
+Usage:
+$ python new_DL2_to_DL3.py -c configuration_file.yaml
+"""
+import argparse
+import glob
+import logging
+import os
+from pathlib import Path
+
+import joblib
+import numpy as np
+import pandas as pd
+import yaml
+
+from magicctapipe import __version__
+from magicctapipe.io import resource_file
+from magicctapipe.scripts.lst1_magic.semi_automatic_scripts.clusters import (
+    rc_lines,
+    slurm_lines,
+)
+
+__all__ = ["configuration_DL3", "DL2_to_DL3"]
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def configuration_DL3(target_dir, source_name, config_file, ra, dec):
+    """
+    This function creates the configuration file needed for the DL2 to DL3 conversion
+
+    Parameters
+    ----------
+    target_dir : str
+        Path to the working directory
+    source_name : str
+        Source name
+    config_file : str
+        Path to MCP configuration file (e.g., resources/config.yaml)
+    ra : float
+        Source RA
+    dec : float
+        Source Dec
+    """
+
+    if config_file == "":
+        config_file = resource_file("config.yaml")
+
+    with open(
+        config_file, "rb"
+    ) as fc:  # "rb" mode opens the file in binary format for reading
+        config_dict = yaml.safe_load(fc)
+    DL3_config = config_dict["dl2_to_dl3"]
+    DL3_config["source_name"] = source_name
+    DL3_config["source_ra"] = f"{ra} deg"
```

Comment:
is the source_ra used anywhere?
I guess not, but well, it is also a useful variable in the database